### DISCUSSION TITLE
custom theme colors don't seem to work

### DISCUSSION THREAD

**Discussion by Danjar27** [nativewind/nativewind#169] [Q&A]
When I define custom colors in **tailwind.config.ts** and try to use them in classNames, they don't work
![image](https://user-images.githubusercontent.com/76726139/184444242-d699e28c-eb7f-4459-a0c8-094d212b9147.png)
if I use default colors or inline colors (like bg-[#fff]) everything seems to work as expected, tho. 
![image](https://user-images.githubusercontent.com/76726139/184444388-fcc3e1de-46be-4bb5-9888-4a5be358b6a7.png)
Is there a way to fix this?

**Comment by marklawlor** [comment]
Please follow the troubleshooting guide https://www.nativewind.dev/guides/troubleshooting.

Please also make sure you are on the latest version of nativewind, as older versions did not work with `DEFAULT` colors

Relevant issue: https://github.com/marklawlor/nativewind/issues/80

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
